packages/phoenix-client/examples/prompts/anthropic_messages.py

```diff
@@ -0,0 +1,25 @@
+import json
+
+from anthropic import Anthropic
+from openinference.instrumentation.anthropic import AnthropicInstrumentor
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import SimpleSpanProcessor
+
+from phoenix.client import Client
+from phoenix.client.helpers.sdk.anthropic.messages import to_kwargs
```

Comment:
```suggestion
from phoenix.client.prompt.anthropic import to_kwargs
```
The import path is really long and feels hard to discover. what do you think about namespacing it under prompt?

Just trying to think about first time discoverability of functions. I think `helpers` might not be the first place I search for.

Alternatively we could encapsulate all anthropic logic under `phoenix.client.anthropic`?

Reply:
provided a single helper that delegates based on `model_provider`